.circleci/config.yml

```diff
@@ -151,25 +151,27 @@ workflows:
           name: build-v<< matrix.node-version >>
           matrix:
             parameters:
-              node-version: [ "16.15", "14.19" ]
+              node-version: [ "18.7", "16.15", "14.19" ]
       - test:
           requires:
             - build-v<< matrix.node-version >>
           name: test-v<< matrix.node-version >>
           matrix:
             parameters:
-              node-version: [ "16.15", "14.19" ]
+              node-version: [ "18.7", "16.15", "14.19" ]
       - lint:
           requires:
             - build-v<< matrix.node-version >>
           name: lint-v<< matrix.node-version >>
           matrix:
             parameters:
-              node-version: [ "16.15", "14.19" ]
+              node-version: [ "18.7", "16.15", "14.19" ]
```

Comment:
Does the steps being run for each Node.js version run in tandem or successively?

I'm trying to gauge if this might potentially slow down our CI process.

If they run in tandem (which I think they do based on how a typical [`build-test` flow](https://github.com/Financial-Times/dotcom-reliability-kit/runs/7838898914) plays out), might running the same steps for an increasing number of Node.js versions somehow else cause bottlenecks (e.g. the number of simultaneous CircleCI containers our plan provides)?

I am raising these points based on some previous Platforms work to try and avoid/reduce the length of CircleCI queues.

Reply:
They run in tandem yep, and yes that could potentially cause bottlenecks if our CircleCI plan doesn't quite cater for it. _however_ I think:

1. testing against multiple stable Node.js versions is something we need and our CI tooling needs to handle that. If it's an issue we could either try and get more capacity or move to GitHub actions which would be free (and not impact our quota) as this is an open source repo
2. [Node.js 14 end-of-life is April 2023](https://nodejs.org/en/about/releases/) so we'll need increased capacity between October 2022 and April 2023 at which point we can drop Node.js 14 support

Reply:
Another alternative is we leave this open as a long-running PR and just periodically rebase it and run the tests in Node.js 18, then we merge in October to reduce the window where we need increased capacity

Reply:
October will come soon enough so I have no issues with merging now.